README.md

```diff
@@ -139,6 +142,8 @@ Direwolf on Linux is confirmed working, please report any issues. Note that curr
 While they are annoying, they are safe to ignore and require changes in the upstream direwolf sourcecode. See https://github.com/wb2osz/direwolf/issues/448 for more details.
 
 ## Notes on Direwolf
+(https://github.com/wb2osz/direwolf)
```

Comment:
Instead of adding the URL in as text, consider using a link in the text. See https://github.github.com/gfm/#links